src/libraries/Directory.Build.props

```diff
@@ -328,4 +328,10 @@
     <CLSCompliant Condition="'$(CLSCompliant)' == '' and ('$(IsTestProject)' == 'true' or '$(IsTestSupportProject)' == 'true')">false</CLSCompliant>
     <CLSCompliant Condition="'$(CLSCompliant)' == ''">true</CLSCompliant>
   </PropertyGroup>
+
+  <PropertyGroup>
+    <LibrariesPretrimmedArtifactsPath>$(ArtifactsBinDir)ILLinkTrimAssembly/$(BuildSettings)/pretrimmed</LibrariesPretrimmedArtifactsPath>
```

Comment:
Add a common property for `$(ArtifactsBinDir)ILLinkTrimAssembly/$(BuildSettings)` and use MSBuild::NormalizePath:

```xml
$([MSBuild]::NormalizePath('$(ArtifactsBinDir)', 'ILLinkTrimAssembly', '$(BuildSettings)'))
```